src/main/java/org/maxgamer/quickshop/util/reporter/error/RollbarErrorReporter.java

```diff
@@ -154,20 +156,22 @@ public RollbarErrorReporter(@NotNull QuickShop plugin) {
                     pasteURL = this.lastPaste;
                 }
             }
-            this.rollbar.error(throwable, this.makeMapping(), throwable.getMessage());
-            plugin
-                    .getLogger()
-                    .warning(
-                            "A exception was thrown, QuickShop already caught this exception and reported it, switch to debug mode to see the full errors.");
-            plugin.getLogger().warning("====QuickShop Error Report BEGIN===");
-            plugin.getLogger().warning("Description: " + throwable.getMessage());
-            plugin.getLogger().warning("Server   ID: " + plugin.getServerUniqueID());
-            plugin.getLogger().warning("====QuickShop Error Report E N D===");
-            Util.debugLog(throwable.getMessage());
-            Arrays.stream(throwable.getStackTrace()).forEach(a -> Util.debugLog(a.getClassName() + "." + a.getMethodName() + ":" + a.getLineNumber()));
-            if (Util.isDevMode()) {
-                throwable.printStackTrace();
-            }
+            new Thread(() -> {
```

Comment:
Why we can't just print it?

Reply:
I want to hide errors from the user side because we can see them.
Also, use thread because this is the main-thread working and it stuck on my server once.